File: 1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.js
/**
 * @param {number} m
 * @param {number} n
 * @param {number[][]} indices
 * @return {number}
 */
var oddCells = function(m, n, indices) {
  const mat = Array(m).fill().map(_ => Array(n).fill(0))
  indices.forEach(([r, c]) => {
    for (let i = 0; i < n; i++) mat[r][i] += 1
    for (let i = 0; i < m; i++) mat[i][c] += 1
  })
  let ans = 0
  mat.forEach(r => r.forEach(c => ans += c % 2))
  return ans
};

File: 1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.go
func oddCells(m int, n int, indices [][]int) int {
  mat := make([][]int, m)
  for i := range mat {
    mat[i] = make([]int, n)
  }
  for _, v := range indices {
    r, c := v[0], v[1]
    for i := 0; i < n; i++ {
      mat[r][i] += 1
    }
    for j := 0; j < m; j++ {
      mat[j][c] += 1
    }
  }
  ans := 0
  for _, r := range mat {
    for _, v := range r {
      if v % 2 == 1 {
        ans++
      }
    }
  }
  return ans
}